extensions/blocks/calendly/calendly.php

```diff
@@ -115,11 +115,25 @@ function load_assets( $attr, $content ) {
 				esc_attr( $block_id )
 			);
 			$script  = <<<JS_END
-Calendly.initInlineWidget({
-	url: '%s',
-	parentElement: document.getElementById('%s'),
-	inlineStyles: false,
-});
+// Initialize function as an array object
+// to allow for multiple widgets.
+var jetpackInitCalendly = [
```

Comment:
I tried using the unique `$block_id` to assign each function to a unique object key, but that got complicated, especially if we wanted to escape it. Alternatively we could print out the `Calendly.initInlineWidget` call twice, instead of assigning it to a function. But then we'd need to double the inserted strings in `sprintf...`.